src/valgrind.supp

```diff
@@ -27,6 +27,16 @@
    fun:lzf_compress
 }
 
+{
+   <False positive on CircleCI>
```

Comment:
I don't know why this is necessary, but Circle reports a leak on https://github.com/RedisGraph/RedisGraph/pull/1388/files#diff-04b07b6c04f2ed4b6fa0675c536adefd597edc5690b0d7a92f1edaae758a39dbR134. I don't see how this is possible and cannot reproduce it myself.